extensions/objects/templates/ratio-expression-editor.component.ts

```diff
@@ -28,18 +28,21 @@ import { downgradeComponent } from '@angular/upgrade/static';
 })
 
 export class RatioExpressionEditorComponent implements OnInit {
-  @Input() modalId: symbol;
-  @Input() value;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() modalId!: symbol;
+  @Input() value!: number[];
+  localValue!: { label: string; };
   @Output() valueChanged = new EventEmitter();
   warningText: string = '';
-  localValue: { label: string; };
   eventBusGroup: EventBusGroup;
   constructor(private eventBusService: EventBusService) {
     this.eventBusGroup = new EventBusGroup(this.eventBusService);
   }
 
   ngOnInit(): void {
-    if (this.value === null) {
+    if (this.value === undefined) {
```

Comment:
Why is this needed?

Reply:
`value` is undefined at the top and comparing with null doesn't seem appropriate